hadoop-ozone/dist/src/main/compose/testlib.sh

```diff
@@ -201,6 +201,10 @@ execute_robot_test(){
 
   copy_daemon_logs
 
+  if [[ ${rc} -gt 0 ]] && [[ ${rc} -lt 250 ]]; then
```

Comment:
Just curious, why do we exclude exit codes >= 250?

Reply:
http://robotframework.org/robotframework/3.2.1/RobotFrameworkUserGuide.html#return-codes

On second look, it should be `-le 250`.